internal/provider/resource_gitlab_project_issue_test.go

```diff
@@ -0,0 +1,211 @@
+package provider
+
+import (
+	"fmt"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabProjectIssue_basic(t *testing.T) {
+	testAccCheck(t)
+
+	var testIssue gitlab.Issue
+
+	testProject := testAccCreateProject(t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabProjectIssueDestroy,
+		Steps: []resource.TestStep{
```

Comment:
These tests are a little weak because they do not check what happened upstream. So this will not catch bugs resulting from not passing attributes to the upstream API and not setting the attribute in the READ function.

Reply:
As discussed on discord :D